Lib/test/test_bytes.py

```diff
@@ -1382,6 +1382,19 @@ def test_bytearray_api(self):
             except OSError:
                 pass
 
+    def test_mod_use_after_free(self):
```

Comment:
I would refer to mutation in the test name. Use-after-free is just one of possible effects of the bug. The cause is mutation during formatting.